Drop-In/src/main/java/com/braintreepayments/api/DropInClient.java

```diff
@@ -327,11 +334,18 @@ void getSupportedPaymentMethods(final FragmentActivity activity, final GetSuppor
     void getSupportedCardTypes(final GetSupportedCardTypesCallback callback) {
         braintreeClient.getConfiguration((configuration, error) -> {
             if (configuration != null) {
-                List<String> supportedCardTypes = new ArrayList<>(configuration.getSupportedCardTypes());
+                Set<CardType> supportedCardTypes = new HashSet<>();
```

Comment:
Since we are converting supportedCardTypes to an ArrayList to return it, can we just initialize it as an ArrayList here instead?

Reply:
Yeah good call. I did that to filter out duplicates, but thinking about it `parseCardType` returns `null` for `DropInPaymentMethod.UNKNOWN`. Unless it's duplicated in the gateway that scenario won't happen.